add notes on how to deploy retirement pages

camallen opened the pull request:
add some docs for future folks about how these pages should be deployed.

camallen committed:
add notes on how to deploy retirement pages

zwolf reviewed (commented):
This will likely cause file collisions (i.e. two index.html files). I have been renaming the existing one to something else (index.html.old or w/e), then copying the other contents in. I presume that the CLI will just overwrite automatically, but even if it fails this should be noted here.

The GUI is a pretty good choice for this, maybe put that at the top.

mcbouslog reviewed (approved):
The aws cli instructions worked and the GUI looked like it would have been easy to use as well, both great options.

Maybe mention to see changes immediately might have to clear the static sites cache (only way I know is in Slack with `lita clear static cache`)?

adammcmaster commented:
> This will likely cause file collisions (i.e. two index.html files). I have been renaming the existing one to something else (index.html.old or w/e), then copying the other contents in. I presume that the CLI will just overwrite automatically, but even if it fails this should be noted here.

It's worth mentioning that the Zooniverse-static bucket does have versioning enabled, so an accidental overwrite can be reverted (not sure how easy that would actually be in practice if a lot of files were overwritten though).

camallen committed:
add notes on static cache and index preservation

reword the docs adding index preservation bash cmds and cmds to clear the static cache.
